make/langtools/netbeans/nb-javac/src/META-INF/upgrade/nbjavac.hint

```diff
@@ -337,3 +355,36 @@ sun.reflect.annotation.AnnotationTypeMismatchExceptionProxy
 nbjavac.sun.annotation.AnnotationTypeMismatchExceptionProxy
 ;;
 
+$cs.isEmpty() :: $cs instanceof java.lang.CharSequence
+=>
+($cs.length() == 0)
+;;
+
+$coll.toArray($type[]::new) :: inClass("com.sun.tools.javac.comp.Flow")
```

Comment:
This is probably needed in order to run on JDK-8, right? The [toArray(IntFunction)](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Collection.html#toArray(java.util.function.IntFunction)) has only been added in JDK-11.


Why there is a restriction `inClass("com.sun.tools.javac.comp.Flow")` - this seems like a legitimate generic rule to use.

Reply:
You are right. Fixed.